lte/gateway/c/core/oai/lib/message_utils/service303_message_utils.h

```diff
@@ -26,4 +26,9 @@
 int send_app_health_to_service303(
     task_zmq_ctx_t* task_zmq_ctx_p, task_id_t origin_id, bool healthy);
 
+int send_stats_to_service303(
+    task_zmq_ctx_t* task_zmq_ctx_p, task_id_t origin_id,
+    uint32_t nb_enb_connected, uint32_t nb_ue_attached,
+    uint32_t nb_ue_connected);
```

Comment:
can you actually use the structure that you defined? It will be easier to extend when we want to pass more stats/states in the future.

Reply:
Good point, I have a follow-up change that converts this to a C++ object. Can I put a pin on this and get back within a week?
Technically if it is a C object we could reuse application_mme_stats_msg, so if you would like to see a struct here, I can replace this interface with that as well.

Reply:
The underlying reason for this change apart from being right is to convert the dataobjects in mme_app to C++ and that would require service303 not reaching out into the mme_app as it doesn't compile as I am not migrating service303 thread.

Reply:
let's reuse application_mme_stats_msg to pack the parameters now, as I only see the benefits of doing so.

Reply:
Updated with application_mme_stats_msg_t for now. As noted I am still figuring out the right way to replace the union, will float out a proposal to support a polymorphic ITTI msg object in C++. Thanks for the review